### DISCUSSION TITLE
4/17 (월) 제이미 - Configuration과 TestConfiguration 빈 등록 시 충돌 해결 방법

### DISCUSSION THREAD

**Discussion by JJ503** [woowacourse-study/2023-presentation-study#3] [발표 피드백]
[자료](https://treasure-fountain-650.notion.site/Configuration-TestConfiguration-35cf973bea2542e48ad96ddbec7fed1d)

**Comment by sh111-coder** [comment]
* 설명을 조리있게 잘 해준 것 같아요!
* 발표 속도랑 톤이 좋았습니다!
* 경험해본 트러블을 해결한 주제라서 더 자신있게 느껴졌어요

**Comment by jyeost** [comment]
- 발표 성량, 속도, 발음 모두 적절했습니다!
- 코드 예시가 많아서 어려운 주제임에도 이해하기 쉬웠습니다
- 실제로 겪은 문제를 해결해 나가는 발표 주제를 선정한 것도 좋았습니다 :)
- 중반 이후부터 말을 더듬거나, 가끔씩 버퍼링이 걸리는 것이 아쉬웠습니다.

**Comment by yeonkkk** [comment]
* 발음도 명확하고, 발표 주제를 온전히 이해한 상태에서 설명해주시는 것 같다고 느꼈습니다.
* 발표 속도랑 자료 분량이 적절했던 것 같습니다.
* 예시가 많아서 이해하기 좋았습니다.
* 가끔 문장이 마무리가 안 될 때가 있었던 것 같아요!

**Comment by amaran-th** [comment]
- 어려운 내용이었는데 잘 정리해주셔서 귀에 쏙쏙 들어왔어요!
- 목소리가 좋으셔서 집중이 잘 됐습니다 (발표에 적합한 톤과 속도)
- 예제 코드에 신경 많이 쓰신 게 느껴져요

**Comment by kwonyj1022** [comment]
* 예시 코드가 있어서 좋았습니다.
* 주제가 어려웠어서 중간중간 청중이 이해할 수 있도록 잠깐 쉬는 타임이 있으면 좋을 것 같습니다. (생각하고 있는 중간에 다음 설명이 진행되지 않도록)
